astronomer/providers/amazon/aws/example_dags/example_emr_job_flow_manual_steps.py

```diff
@@ -0,0 +1,104 @@
+"""Example DAG's for AWS EMR related operator and sensor"""
+
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.models.baseoperator import chain
+from airflow.providers.amazon.aws.operators.emr import (
+    EmrAddStepsOperator,
+    EmrCreateJobFlowOperator,
+    EmrTerminateJobFlowOperator,
+)
+
+from astronomer.providers.amazon.aws.sensors.emr import EmrStepSensorAsync
+
+JOB_FLOW_ROLE = os.environ.get("EMR_JOB_FLOW_ROLE", "EMR_EC2_DefaultRole")
+SERVICE_ROLE = os.environ.get("EMR_SERVICE_ROLE", "EMR_DefaultRole")
+AWS_CONN_ID = os.environ.get("ASTRO_AWS_CONN_ID", "aws_default")
+EMR_CONN_ID = os.environ.get("ASTRO_EMR_CONN_ID", "emr_default")
+
+
+SPARK_STEPS = [
+    {
+        "Name": "calculate_pi",
+        "ActionOnFailure": "CONTINUE",
+        "HadoopJarStep": {
+            "Jar": "command-runner.jar",
+            "Args": ["/usr/lib/spark/bin/run-example", "SparkPi", "10"],
+        },
+    }
+]
+
+JOB_FLOW_OVERRIDES = {
+    "Name": "PiCalc",
+    "ReleaseLabel": "emr-5.29.0",
+    "Applications": [{"Name": "Spark"}],
+    "Instances": {
+        "InstanceGroups": [
+            {
+                "Name": "Primary node",
+                "Market": "ON_DEMAND",
+                "InstanceRole": "MASTER",
+                "InstanceType": "m4.large",
+                "InstanceCount": 1,
+            },
+        ],
+        "KeepJobFlowAliveWhenNoSteps": False,
+        "TerminationProtected": False,
+    },
+    "JobFlowRole": JOB_FLOW_ROLE,
+    "ServiceRole": SERVICE_ROLE,
+}
+
+default_args = {
+    "execution_timeout": timedelta(minutes=30),
+    "trigger_rule": "all_done",
```

Comment:
Are we using these trigger values like `all_done` across other modules too? In that case, better to make it a constant and reuse so that in case we choose to change such static values later it is easier to do in one place rather than changing it one by one in all modules.